packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/ListView/tests/index.test.js

```diff
@@ -116,16 +116,48 @@ describe('ADMIN | Pages | TRANSFER TOKENS | ListPage', () => {
     });
 
     expect(container.firstChild).toMatchInlineSnapshot(`
```

Comment:
IMO we should refactor this test, we now have a 500 line snapshot that does not tell us _anything_. We should be asserting pieces of information for the user are there.

Reply:
removed the snapshot